### ISSUE TITLE
Change Previous/Next buttons on now playing screen

### ISSUE THREAD

**Issue by campbellkerr** [TeamNewPipe/NewPipe#3869]
#### Describe the feature you want
When playing a video full screen, there are 3 main controls - Previous, Play/Pause and Next. These are mostly redundant unless you're watching multiple videos in a playlist. In fact, the Next button is available but does nothing when watching a single video, and the Previous button goes to the start of the video (and is very easy to accidentally hit).

![image](https://user-images.githubusercontent.com/38451588/87004225-6fd15600-c200-11ea-85ec-5bf8aaa1b768.png)

Spotify and some other audiobook/podcast apps have a nicer -15sec/+15sec control instead which is great for long durations.

![image](https://user-images.githubusercontent.com/38451588/87004980-a52a7380-c201-11ea-88f8-3d842736af37.png)

Another option could be to make the buttons fast forward/rewind when holding them down.

#### How will you/everyone benefit from this feature?

Less accidentally starting my video over, way easier way to rewind a video by a small amount instead of having to drag the "scrubber" inaccurately. Better use of screen real estate, more useful functions!

**Comment by B0pol** [comment]
I agree for single videos it should be changed. Plus it would maki it easy for Android TV to browse the video (it's impossible to double tap).

But for the icon, I remind you that you can change the duration in settings, therefore the icon need to be dynamic if possible or make 5 icons?

**Comment by Stypox** [comment]
> But for the icon, I remind you that you can change the duration in settings, therefore the icon need to be dynamic if possible or make 5 icons?

The same icons used in the background player could be used. Also I would add those icons only to Android TV users to fast-forward/-rewind, since for other users there is already the double tap gesture

**Comment by SyCoREAPER** [comment]
@SameenAhnaf

If you feel this merge is more appropriate that is of course fine.

However I completely disagree with the design proposal of this submission.
